runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -4034,6 +4039,8 @@ bool
 JITaaSHelpers::shouldRetryConnection(OMRPortLibrary *portLibrary)
    {
    OMRPORT_ACCESS_FROM_OMRPORT(portLibrary);
+   if (feGetEnv("TR_JITServerAssertStreamFailure"))
```

Comment:
What is the consequence of not having this code?

Reply:
consequence of not having this code is that every test that follows the problematic test will also fail due to the crashed server. Which means 1. we get a ton of useless core dumps 2. we might not be able to tell which test actually crashed the server.

Reply:
let's add this explanation as a comment then